exception/ExceptionHandler.js

```diff
@@ -58,6 +64,16 @@ export class ExceptionHandler {
     //--------------------------------
     // Implementation
     //--------------------------------
+    static hideParams(requestOptions, params) {
+        delete requestOptions.body;
+
+        if (!requestOptions.params) return;
+
+        params.forEach(param => {
+            delete requestOptions.params[param];
```

Comment:
would it be better if we just set the parameter to ‘*****’
otherwise somebody troubleshooting might think its just missing